src/screens/MixpanelHook.res

```diff
@@ -97,3 +92,94 @@ let useSendEvent = () => {
     }
   }
 }
+
+let usePageView = () => {
+  open GlobalVars
+  open Window
+  let fetchApi = AuthHooks.useApiFetcher()
+  let {getUserInfoData} = React.useContext(UserInfoProvider.defaultContext)
+  let environment = GlobalVars.hostType->getEnvironment
+  let {clientCountry} = HSwitchUtils.getBrowswerDetails()
+  let country = clientCountry.isoAlpha2->CountryUtils.getCountryCodeStringFromVarient
+  let featureFlagDetails = HyperswitchAtom.featureFlagAtom->Recoil.useRecoilValueFromAtom
+  async (~path) => {
+    let mixpanel_token = Window.env.mixpanelToken
+    let {email, merchantId, name} = getUserInfoData()
+    let body = {
+      "event": "page_view",
+      "properties": {
+        "token": mixpanel_token,
+        "distinct_id": email,
+        "$device_id": email->String.split(":")->Array.get(1),
+        "$screen_height": Screen.screenHeight,
+        "$screen_width": Screen.screenWidth,
+        "name": email,
+        "merchantName": name,
+        "email": email,
+        "mp_lib": "restapi",
+        "merchantId": merchantId,
+        "environment": environment,
+        "lang": Navigator.browserLanguage,
+        "$os": Navigator.platform,
+        "$browser": Navigator.browserName,
+        "mp_country_code": country,
+        "page": path,
+      },
+    }
+
+    try {
+      if featureFlagDetails.mixpanel {
+        let _ = await fetchApi(
+          `${getHostUrl}/mixpanel/track`,
+          ~method_=Post,
+          ~bodyStr=`data=${body->JSON.stringifyAny->Option.getOr("")->encodeURI}`,
+        )
+      }
+    } catch {
+    | _ => ()
```

Comment:
Here as well

Reply:
we will ignore.we don't want to capture the mixpanel errors